kythe/release/BUILD

```diff
@@ -69,6 +70,7 @@ genrule(
         "$(locations misc)",
         "--cp $(location release.BUILD) BUILD",
         "--cp $(location release.WORKSPACE) WORKSPACE",
+        "--cp $(location external.bzl) ./",
```

Comment:
we should ensure that these are kept in sync with the Kythe dependencies, rather than being duplicated.

Reply:
What's the best way to keep things in sync? I could change the main repo's external.bzl to pull in this new one, but I think it gets a little hairy with some of the order-dependent calls to things like `rules_proto_dependencies()`, `rules_proto_toolchains()`, `protobuf_deps()`.

Reply:
I think the easiest way is to source this (or an equivalent) file from //:external.bzl and call `kythe_release_dependencies` from there. Likely with a comment around what sorts of things it pulls in.